### ISSUE TITLE
[ACCESSIBILITY] Add Alternative texts for control elements

### ISSUE THREAD

**Issue by markusmann-vg** [hzi-braunschweig/SORMAS-Project#9635]
### Feature Description

There are no text alternatives for information-bearing icons. The icons are included as ASCII characters of an icon font.

Examples:
- Edit person (Location coordinates area)
- Dashboard (calendar icons, fullscreen icons)

![Bildschirmfoto 2022-06-27 um 15 14 26](https://user-images.githubusercontent.com/62540178/175950325-cbc5676e-6326-480a-b78e-8e7adeaf6644.png)

Icons should be included as pseudo elements. 

If the icon is an information-carrying icon without visually visible text (stand-alone icon), a text alternative should be available. This can be, for example, text that is hidden using a suitable method (not display:none) or that is provided via an aria-label. It is useful to hide the icon itself with a suitable technique for screen readers (e.g. aria-hidden="true").

### Proposed Change
Add alternative text (has to be intenationalizable via crowdin) to every information-carrying icon

### Acceptance Criteria
Using the browser's Web Developer Tools, check whether content (font icons) is included with the CSS property content for the :before or :after pseudo elements.

Check if the icon is hidden with an appropriate technique for screen readers (e.g. aria-hidden="true").

For information-carrying icons, check whether an HTML text alternative is available that is made invisible via CSS.

If there are HTML text alternatives that are not visible on the screen: Check what method was used to hide them. If display:none is used, this is to be evaluated like a non-existent or empty alt attribute.

If an appropriate CSS technique was used: Check if the text link is an equivalent text alternative for the icon (see 2.6 Appropriate alternative text).

If no alternative text is available via accessible hidden or co-linked text, check whether the text alternative is provided via a title attribute or aria-label.

If text is output for these icons (e.g. content: "k"), check if the icon is hidden using an appropriate technique for screen readers (e.g. aria-hidden="true").
